shell/testing/BUILD.gn

```diff
@@ -36,3 +36,40 @@ executable("testing") {
     "//third_party/skia",
   ]
 }
+
+# A second tester binary with SUPPORT_FRACTIONAL_TRANSLATION enabled.
+executable("testing_v2") {
```

Comment:
please name this something clearer. Something like

`testing_fractional_translation`.

Reply:
Done